Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -582,7 +578,7 @@ extension MapView: DelegatingMapClientDelegate {
         metalView.isOpaque = isOpaque
         metalView.layer.isOpaque = isOpaque
         metalView.isPaused = true
-        metalView.enableSetNeedsDisplay = true
+        metalView.enableSetNeedsDisplay = false
```

Comment:
`enableSetNeedsDisplay` default value is false. I suppose we can just remove it.